index.html

```diff
@@ -0,0 +1,202 @@
+              <div class="card">
+                <div class="img_Bx">
+                  <img src="services/services-item-1.png" alt="#">
+                </div>
+                <div class="content">
+                  <h3>Custom Design</h3>
+                  <p class="card_description">Curabitur vestibulum eget mauris quis laoreet. Phasellus in quam
+                  laoreet, viverra lacus ut, ultrices velit.</p>
+                </div>
+              </div>
+              <div class="card">
+                <div class="img_Bx">
+                  <img src="services/services-item-2.png" alt="#">
+                </div>
+                <div class="content">
+                  <h3>Inovative Ideas</h3>
+                  <p class="card_description">Quisque luctus, quam eget molestie commodo, lacus purus cursus
+                    purus, nec rutrum tellus dolor id lorem.</p>
+                </div>
+              </div>
+              <div class="card">
+                <div class="img_Bx">
+                  <img src="services/services-item-3.png" alt="#">
+                </div>
+                <div class="content">
+                  <h3>Love Is The Answer</h3>
+                  <p class="card_description">Nulla sed nunc et tortor luctus faucibus. Morbi at aliquet
+                    turpis, et consequat felis. Quisque condimentum.</p>
+                </div>
+              </div>
+              <div class="card">
+                <div class="img_Bx">
+                  <img src="services/services-item-4.png" alt="#">
+                </div>
+                <div class="content">
+                  <h3>Responsive Layout</h3>
+                  <p class="card_description">Sed porttitor placerat rhoncus. In at nunc tellus. Maecenas
+                    blandit nunc ligula. Praesent elit leo.</p>
+                </div>
+              </div>
+              <div class="card">
+                <div class="img_Bx">
+                  <img src="services/services-item-5.png" alt="#">
+                </div>
+                <div class="content">
+                  <h3>24 / 7 Support</h3>
+                  <p class="card_description">Vivamus vel quam lacinia, tincidunt dui non, vehicula nisi.
+                    Nulla a sem erat. Pellentesque egestas venenatis lorem.</p>
+                </div>
+              </div>
+              <div class="card">
+                <div class="img_Bx">
+                  <img src="services/services-item-6.png" alt="#">
+                </div>
+                <div class="content">
+                  <h3>Feel Like A Star</h3>
+                  <p class="card_description">Quisque hendrerit purus dapibus, ornare nibh vitae, viverra
+                    nibh. Fusce vitae aliquam tellus.</p>
+                </div>
+              </div>
+            </div>
+          </section>
+       </div>
+
+
+        <div id="portfolio" class="contain_portfolio">
+          <section class="portfolio">

```

Comment:
It is better to set contain_portfolio inside section to get main divided on sections.